src/constants/wallets.ts

```diff
@@ -383,9 +383,10 @@ export const DYDX_CHAIN_INFO: Parameters<typeof suggestChain>[0] = {
   features: [],
 };
 
+// TODO: export this type from abacus instead
 export enum DydxChainAsset {
-  USDC = 'USDC',
-  DYDX = 'Dv4TNT',
+  USDC = 'usdc',
+  DYDX = 'dydx',
```

Comment:
Need to update the TransferForm receipt since it uses these values for display as well.
<img width="125" alt="Screen Shot 2023-09-22 at 11 30 57 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/c578ff9c-9a79-4984-966a-fec717ba843c">

Reply:
oh nice catch!